### PR TITLE
Adding / replacing moves now has an additional screen to show what old/new moves are

### PR DESCRIPTION
**Pull request by magroader**
#427

### CODE REVIEW

File: screens/change_pokemon/move/move_summary.lua [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+local function join_table(title, T, sep)
+	if T then
+		return title .. table.concat(T, sep)
+	end
+	return "-"
+end
+
+local function update_desc_list(str_node_prefix, action_id, action)
+	gooey.vertical_static_list(str_node_prefix, str_node_prefix .. "/desc_stencil", {str_node_prefix .. "/txt_desc"}, action_id, action)
```
[the remark this answers is not in the document]
Comment (magroader) [reply screens/change_pokemon/move/move_summary.lua:17]: I didn't notice any problems... is it spouting some warnings or something?

[the remark this answers is not in the document]
Comment (magroader) [reply screens/change_pokemon/move/move_summary.lua:17]: Oop, yeah, you're right - clicking directly on the description without scrolling throws an error ::facepalm::

**Comment by Jerakin** [comment]
Some feedback on the visual aspect of it
* Remove grey background (to keep the theamatically consistent with the other move view), 
* Removing the bg will make the arrow a lot less visible so maybe make it white (the color language is also orange means it's intractable) and increase the size with 50%. I think we might have to update the arrow grapichs for this as I bet it won't visually hold up for being scaled up like that.
* Make the colored background frame a bit bigger (maybe with 10-15 pixels)
* The screen doesn't scale well (buttons overlap bottom move) on taller screens (you can specify a resolution in the main script, iphonex in this case).

Some more _what if?_ ideas/feedback
* Maybe it would be cool with the image that says "Normal" (used under traits in the party screen) instead of a text?
* It could help with reading that the description can scroll if a scrollbar was there (urgh this reminds me that I still haven't done #107)

**Comment by Jerakin** [comment]
Oh and people will LOVE this feature!

**Comment by magroader** [comment]
This feedback will take me some time to work through.

**Comment by magroader** [comment]
> Maybe it would be cool with the image that says "Normal" (used under traits in the party screen) instead of a text?
Do you mean this part?
image

**Comment by magroader** [comment]
I tried to get the Vulnerabilities to show as icons but had a lot of trouble, so spun off #466

**Comment by Jerakin** [comment]
iirc it said "Grass" (or whichever type the move was) in the top right corner which would be cool to replace with the image that you show in your second image. But replacing that part would be cool too!

**Comment by magroader** [comment]
Oh geez, NOW I understand what you were saying. The icon was already there next to the text, it was just on the wrong layer, wasn't visible, and I didn't notice. I just assumed it was working when you commented on it and thought you were talking about someplace else.

The only remaining outstanding feedback is to put a scrollbar in the description area, and I want to wait for #452 to do that one.

**Comment by magroader** [comment]
This is now ready to go. We decided not to add actual scrollbars to the move descriptions.

**Comment by Jerakin** [comment]
Looking awesome! People have been asking for something like this for _ages_

**Pull request merged**